resources/lib/vrtplayer/vrtplayer.py

```diff
@@ -4,7 +4,7 @@
 
 from __future__ import absolute_import, division, unicode_literals
 from resources.lib.helperobjects.helperobjects import TitleItem
-from resources.lib.vrtplayer import actions
+from resources.lib.vrtplayer import actions, streamservice, tokenresolver
```

Comment:
Why import this at the top, when it may only be needed for playing a stream ?
(Whereas we import vrtplayer for almost everything else)

Reply:
I thought this slowed down playback. Maybe I have to check this again.

Reply:
It may, the very first time. The question is whether we want to import this when we start the addon for the first time (or every consecutive time after another addon has been opened), or whether we want to take this "hit" when we start the first playback.

I would expect it is only for the first playback, but if it is not, we should make these imports global so that it is. To be tested.